app/views/epp/domains/create.xml.builder

```diff
@@ -5,7 +5,8 @@ xml.epp_head do
     end
 
     xml.resData do
-      xml.tag!('domain:creData', 'xmlns:domain' => Xsd::Schema.filename(for_prefix: 'domain-ee')) do
+      xml.tag!('domain:creData',
+               'xmlns:domain' => Xsd::Schema.filename(for_prefix: @schema_prefix)) do
```

Comment:
And what about version? As far as I remember `Xsd::Schema.filename(for_prefix: <some_prefix>))` still returns latest version of a filename.

Reply:
Related #2058